docs/presentations/2024-08-27-CU-Anschutz-DBMI-Retreat/_extensions/quarto-ext/poster/_extension.yml

```diff
@@ -0,0 +1,10 @@
+title: Poster
+author: Carlos Scheidegger
```

Comment:
Will this author change or is this the poster template that we are using?

Reply:
Thanks, good question - we began with a template developed by Carlos and I've iterated on it extensively. It may be appropriate to add here, but we didn't develop the extension the template is based on. I'll give this some more thought.

Reply:
Thinking about this a little more, I think I'll leave this as-is, this file pertains to the extension and not the poster template exactly. We made modifications here and the authorship will be present in the modified file via git contributions. In the future we might decide that this is a useful template and distribute our own, at which point, we'd need to update the extension authorship :) .